etc/systemd/nanocurrency.service

```diff
@@ -0,0 +1,13 @@
+[Unit]
+Description=Nano Daemon live network
+After=network.target
+
+[Service]
+Type=simple
+User=nanocurrency
+WorkingDirectory=/var/nanocurrency/RaiBlocks
+ExecStart=/usr/bin/rai_node --daemon
```

Comment:
would it be ok to start moving the data_directory to be Nanocurrency instead of RaiBlocks and then adjust ExecStart with --data_path? but it would be best to decide what we want to move the internal default path to? or will it always stays as ~/RaiBlocks